src/com/minseungkwon/datastructure/Graph/BinaryTree.java

```
@@ -51,5 +51,44 @@ public StringBuilder postorder(Node n, StringBuilder s) {
         }
         return s;
     }
+
+    public void makeTree(T[] array) {
+        this.root = makeTreeR(array, 0, array.length - 1);
+    }
+
+    Node makeTreeR(T[] array, int start, int end) {
+        if (start > end) {
+            return null;
```

Let's throw an exception in this case. :)